### PR TITLE
SDU-1901: Added identify

### PR DESCRIPTION
**Pull request by PaulinaDrozda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1241,6 +1241,7 @@ Type | Interface                          | Value type | Properties | Descriptio
 -----|------------------------------------|------------|------------|------------
 in   | cmd.indicator.set_visual_element   | int_map    | duration   | Requests visual element (led or some other light source) to display information. Key is a name of indicator component and value is actual value to set. Duration property defines how long the indicator should display the information
 in   | cmd.indicator.set_text             | str_map    | duration   | Requests text indicator to display text. Key is name of indicator component and value is text to be displayed by the component. Duration property defines how long the indicator should display the information
+in   | cmd.indicator.identify             | null       |            | Requests device to identify itself by switching on and off several times.
```
Comment (eternalamenity) [inline README.md:1244]: "...by switching a visual indicator on and off..."

**Review by eternalamenity — APPROVED** [review]
Fix one comment and You can merge it ;)

**Review by eternalamenity — APPROVED** [review]

File: technology-adapter-api.md [hunk 2]
```diff
@@ -259,3 +259,17 @@ This will produce a report similar to this:
   "tags": []
 }
 ```
+
+### Identify
+
+Allows the user to identify the hub. Following sequence will be repeated three times - a light will turn on for 600ms then turn off for 200ms.
```
Comment (boguslaw-wojcik) [inline technology-adapter-api.md:265]: The hub or the device?

File: README.md [hunk 3]
```diff
@@ -1241,6 +1241,7 @@ Type | Interface                          | Value type | Properties | Descriptio
 -----|------------------------------------|------------|------------|------------
 in   | cmd.indicator.set_visual_element   | int_map    | duration   | Requests visual element (led or some other light source) to display information. Key is a name of indicator component and value is actual value to set. Duration property defines how long the indicator should display the information
 in   | cmd.indicator.set_text             | str_map    | duration   | Requests text indicator to display text. Key is name of indicator component and value is text to be displayed by the component. Duration property defines how long the indicator should display the information
+in   | cmd.indicator.identify             | null       |            | Requests device to identify itself by switching a visual indicator on and off several times.
```
Comment (boguslaw-wojcik) [inline README.md:1245]: Lack of relevant interface properties section here.

File: technology-adapter-api.md (hunk 2 above)
Comment (PaulinaDrozda) [reply technology-adapter-api.md:265]: The hub, it's a requirement for Z-wave certification as far as I know

File: README.md (hunk 3 above)
Comment (PaulinaDrozda) [reply README.md:1245]: Added table

**Review by boguslaw-wojcik — APPROVED** [review]

**Pull request merged**